rs/consensus/src/consensus/block_maker.rs

```diff
@@ -172,7 +173,7 @@ impl BlockMaker {
         height: Height,
```

Comment:
(bloody github doesn't allow me to comment on lines 167-168...)
nit: maybe it makes sense to expand the comment above and mention that we ignore blocks from equivocating block makers